test/cuda/test_execution_engine.cc

```diff
@@ -235,6 +235,24 @@ def convolution(float(N,C,H,W) I, float(O,C,KH,KW) W1, float(O) B)
   checkRtol(diff, inputs, C * KW * KH, 5e-7);
 }
 
+TEST_F(ATenCompilationUnitTest, LocalSparseConvolution) {
+  at::Tensor I = at::CUDA(at::kFloat).rand({N, C, H, W});
+  at::Tensor W1 = at::CUDA(at::kFloat).rand({O, KC, KH, KW});
+  std::vector<at::Tensor> inputs = {I, W1};
+  std::vector<at::Tensor> outputs;
+
+  Check(
+      R"(
+def local_sparse_convolution(float(N, C, H, W) I, float(O, KC, KH, KW) W1) -> (O1) {
+    O1(n, o, h, w) +=! I(n, kc % c, h + kh, w + kw) * W1(o, kc, kh, kw) where c in 0:C
+}
+    )",
+      "local_sparse_convolution",
+      tc::CudaMappingOptions::makeConvolutionCudaMappingOptions(),
+      inputs,
+      outputs);
+}
```

Comment:
Please add an actual check function (even if you have to write it using plain C loops).